src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -172,35 +157,74 @@ class BaselinesTable extends Component {
         return table;
     }
 
-    createModalRows() {
-        const { baselineTableData } = this.props;
+    onSingleSelect = (_, event) => {
+        const { selectOneBaseline, isBaselineSelected } = this.props;
+        let id = event.currentTarget.id;
+
+        selectOneBaseline(id);
+        isBaselineSelected(id);
+    }
+
+    renderRadioButton = (data) => {
+        return (
+            <React.Fragment>
+                <Radio
+                    aria-label={ 'radio' + data[1] }
+                    isChecked={ data.selected }
+                    onChange={ this.onSingleSelect }
+                    name={ data[1] }
+                    id={ data[0] }
+                />
+            </React.Fragment>
+        );
+    }
+
+    createModalRows(modalData) {
```

Comment:
The table component shouldn't really need to know whether it's in a modal or not. So my assumption is that this is capturing a concept related to the Modal's use case but is bleeding the 'modal' terminology into the component. What is the functionality of the table itself that this is associated to? (e.g. single select)